war/src/main/less/base/style.less

```diff
@@ -789,6 +789,7 @@ LABEL.attach-previous {
   padding: 1em;
   margin-bottom: 1em;
   word-break: break-word;
+  font-size: @font-size-sm !important;
```

Comment:
We shouldn't be using important on base styles, why is this needed? Which places needs overriding?